src/driver/postgres/PostgresDriver.ts

```diff
@@ -922,11 +921,12 @@ export class PostgresDriver implements Driver {
      */
     protected async createPool(options: PostgresConnectionOptions, credentials: PostgresConnectionCredentialsOptions): Promise<any> {
 
-        credentials = Object.assign({}, credentials, DriverUtils.buildDriverOptions(credentials)); // todo: do it better way
```

Comment:
Dropping this line drops something specifically for Oracle & the URL parsing which we don't want anyway.

Sounds good to me!